frontend/src/game/game.pug

```diff
@@ -0,0 +1,39 @@
+extends ../tcg
```

Comment:
Why are we *really* using `.pug` for? It seems a little bit more readable, but I'm afraid it's less generic that `.html` and it doesn't seem to improve anything that much.

Reply:
mainly to have things like `extends` and `include` which allow us to keep things more organized instead of requiring all HTML to be on a single file (which could get very ugly as `src/game` gets more complex)

Reply:
There are other alternatives, but this one had a funny name ¯\\_(ツ)_/¯

Reply:
Not much ago we make a huge change, removing vue. I'm afraid we are now adding new stuff, like vue, that we might want to remove later.

Reply:
The problem with vue was mainly that we had added it before we needed it. I even made a comment on how we possibly might actually reinclude it later if it was actually needed.

These changes mainly solve two things:
1. More readable and easy to work with Javascript (especially regarding requires and fetching data from backend)
2. More readable and easy to work with HTML (we can now split files to organize them and no longer require copy-pasting a bunch of stuff!)

Reply:
Also, I am completely open to using a different alternative for either problem, although `webpack` alternatives are kinda iffy.

But I can make a list of alternatives to `pug` if you prefer 🙃